bukkit/src/main/java/com/leonardobishop/quests/bukkit/tasktype/type/SmithTaskType.java

```diff
@@ -0,0 +1,77 @@
+package com.leonardobishop.quests.bukkit.tasktype.type;
+
+import com.google.common.collect.HashBasedTable;
+import com.google.common.collect.Table;
+import com.leonardobishop.quests.bukkit.BukkitQuestsPlugin;
+import com.leonardobishop.quests.bukkit.item.QuestItem;
+import com.leonardobishop.quests.bukkit.tasktype.BukkitTaskType;
+import com.leonardobishop.quests.bukkit.util.TaskUtils;
+import com.leonardobishop.quests.common.player.QPlayer;
+import com.leonardobishop.quests.common.player.questprogressfile.TaskProgress;
+import com.leonardobishop.quests.common.quest.Quest;
+import com.leonardobishop.quests.common.quest.Task;
+import org.bukkit.Material;
+import org.bukkit.entity.Player;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.EventPriority;
+import org.bukkit.event.inventory.SmithItemEvent;
+import org.bukkit.inventory.ItemStack;
+
+public class SmithTaskType extends BukkitTaskType {
+
+    private final BukkitQuestsPlugin plugin;
+    private final Table<String, String, QuestItem> fixedQuestItemCache = HashBasedTable.create();
+
+    public SmithTaskType(BukkitQuestsPlugin plugin) {
+        super("smith", TaskUtils.TASK_ATTRIBUTION_STRING, "Smith a specific item.");
```

Comment:
```suggestion
        super("smithing", TaskUtils.TASK_ATTRIBUTION_STRING, "Smith a specific item.");
```

Could the task type be named `smithing` instead? I realise there isn't exactly a standard but most are named `-ing`.